src/predicates/propPathEq.js

```diff
@@ -0,0 +1,35 @@
+/** @license ISC License (c) copyright 2017 original and current authors */
+/** @author Karthik Iyengar (karthikiyengar) */
+
+const curry = require('../core/curry')
+const isArray = require('../core/isArray')
+const isInteger = require('../core/isInteger')
+const isNil= require('../core/isNil')
+const isString = require('../core/isString')
+const equals = require('../core/equals')
+const Maybe = require('../core/Maybe')
+
+const { Just, Nothing } = Maybe
+
+const lift = x => (!isNil(x) ? Just(x) : Nothing())
+
+// propPathEq :: [String | Number] -> a -> Object -> Boolean
+function propPathEq(path, key, target) {
+  if (!isArray(path)) {
+    throw new TypeError(
```

Comment:
I ❤️ this style choice. Will be stealing that from you!!

Reply:
I'd have loved to take credit, but that's just `prettier` at work. I think it would be a good idea to have `prettier` set up as a pre-commit hook (it integrates with `eslint` too). I'd be happy to take that on if you think it makes sense.